### PR TITLE
Xenos have the choice to build smaller and cheaper silos

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Xenos can build small silo, which are only 600 points but produce half as much larva and have half as much HP.

## Why It's Good For The Game

This is to catter with low pop games, where xenos have a lot of issues to make more than one silo. Having only one target to defend is hard, and you can't afford to lose one pool. With this, xenos have the option to make more pools, at the price of having more place to defend at once

## Changelog
:cl:
add: Xenos can build small pools. Half the price for half the HP and half the larva output. Click on build xenos ability to switch silo type
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/silo.dm [hunk 1]
```diff
@@ -3,13 +3,13 @@ SUBSYSTEM_DEF(silo)
...
-	///How many larva points each pool gives per minute with a maturity of zero
-	var/base_larva_spawn_rate = 0.5
+	///A boost in larva spawn rate, changed when hijacking
+	var/larva_rate_boost = 1
 	///How many larva points are added every minutes in total
 	var/current_larva_spawn_rate = 0
 	///How many psych points one corrupted generator gives
 	var/corrupted_gen_output
-
+
```
Comment (TiviPlus) [inline code/controllers/subsystem/silo.dm:12]:
```suggestion

```

File: code/modules/mob/living/carbon/xenomorph/abilities.dm [hunk 2]
```diff
@@ -1018,12 +1018,20 @@
...
 	/// Pyschic point cost
 	var/psych_cost = SILO_PRICE
 
+/datum/action/xeno_action/activable/build_silo/action_activate()
+	var/mob/living/carbon/xenomorph/X = owner
+	if(X.selected_ability == src)
+		build_small_silo = !build_small_silo
+		var/silo_type = build_small_silo ? "small" : "regular"
+		to_chat(X, "<span class ='notice'> You will now build a [silo_type] silo </span>")
+	. = ..()
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/abilities.dm:1034]:
```suggestion
	return ..()
```

File: code/modules/xenomorph/xeno_structures.dm [hunk 3]
```diff
@@ -32,6 +32,7 @@
 	bound_width = 96
 	bound_height = 96
 	max_integrity = 1000
+	var/larva_spawn_rate = 0.5
```
Comment (TiviPlus) [inline code/modules/xenomorph/xeno_structures.dm:35]: Autodoc

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/xenomorph/xeno_structures.dm [hunk 4]
```diff
@@ -162,6 +163,13 @@
 	if(associated_hive)
 		silos += src
 
+/obj/structure/resin/silo/small_silo
+	name = "small resin silo"
+	icon_state = "weed_silo"
+	desc = "A slimy, oozy resin bed filled"
```
Comment (TiviPlus) [inline code/modules/xenomorph/xeno_structures.dm:169]: Filled with what

Comment (BraveMole) [reply code/modules/xenomorph/xeno_structures.dm:169]: tear of marines?

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**